ts/client/src/client.ts

```diff
@@ -2642,6 +2688,56 @@ export class MangoClient {
       .instruction();
   }
 
+  async perpSettleAll(
+    client: MangoClient,
+    group: Group,
+    mangoAccount: MangoAccount,
+    allMangoAccounts?: MangoAccount[],
+  ): Promise<TransactionSignature> {
+    if (!allMangoAccounts) {
+      allMangoAccounts = await client.getAllMangoAccounts(group, true);
+    }
+
+    const ixs = new Array<TransactionInstruction>();
+    // This is optimistic, since we might find the same opponent candidate for all markets,
+    // and they have might not be able to settle at some point due to safety limits
+    // Future: correct way to do is, to apply the settlement on a copy and then move to next position
+    for (const pa of mangoAccount.perpActive()) {
+      const pm = group.getPerpMarketByMarketIndex(pa.marketIndex);
+      const candidates = pm.getSettlePnlCandidates(
+        client,
+        group,
+        allMangoAccounts,
+        pa.getEquityUi(pm) > 0 ? 'negative' : 'positive',
```

Comment:
is `getEquityUi` the right function for checking if there's settleable pnl?

Suggest giving a name to pa.getEquityUi(pm) > 0, since it pops up often

What if there's no settleable pnl (unlikely, I know)

Reply:
Whis is what the ui did which I just blindly copy pasted, oof, replaced with 
![image](https://github.com/blockworks-foundation/mango-v4/assets/89031858/75466eca-d698-4e81-8a4e-702b38a8b605)

Reply:
up here it still says getEquity